.gitignore

```diff
@@ -1,3 +1,6 @@
-
-.idea/
-.DS_Store/
+.idea/encodings.xml
```

Comment:
You don't need to update .gitignore. `.idea/` will cover all the unnecessary files that are there within this folder. Please remove it.

Reply:
Sure. Changes are reverted back, so resolving this conversation.